VATSpy.dat

```diff
@@ -10650,7 +10650,7 @@ RKSD|Maesanri|37.35|127.26639||RKRR|0
 RKSF|Seoul-Yongdungp'O Rokaf Wc|37.5|126.93306||RKRR|0
 RKSG|Pyongtaek Ab|36.96|127.033333||RKRR|0
 RKSH|Command Post Tango|37.51667|126.98306||RKRR|0
-RKSI|Incheon Intl|37.463333|126.440000||RKRR|0
+RKSI|Incheon Intl|37.463333|126.440000|RKSS|RKRR|0

```

Comment:
There is already have an airport entry for RKSS.

RKSS|Gimpo Intl|37.556944|126.797500||RKRR|0

Reply:
I know, this is just to connect the RKSS approach to the RKSI airport, as it covers this one as well